exasol/analytics/schema/column.py

```diff
@@ -104,62 +91,72 @@ def get_class(cls, sql_name: str) -> type["Column"]:
             return next(c for c in classes if sql_name in c.sql_names())
         except StopIteration:
             raise UnsupportedSqlType(
-                f'Couldn\'t find a subclass of Column for SQL type name "{sql_name}" '
+                f'Couldn\'t find a subclass of ColumnType for SQL type name "{sql_name}" '
             )
 
     @classmethod
     def from_pyexasol(
         cls,
-        column_name: str,
         pyexasol_args: dict[str, Any],
-    ) -> "Column":
+    ) -> "ColumnType":
         sql_type_name = pyexasol_args[PyexasolOption.TYPE.value]
-        column_class = cls.get_class(sql_type_name)
+        subclass = cls.get_class(sql_type_name)
         sql_type = SqlType.from_pyexasol(
             pyexasol_args,
-            column_class.pyexasol_mapping(),
+            subclass.pyexasol_mapping(),
         )
-        return column_class.from_sql(column_name, sql_type)
+        return subclass.from_sql(sql_type)
 
     @classmethod
-    def from_sql_spec(cls, column_name: str, spec: str) -> "Column":
+    def from_sql_spec(cls, spec: str) -> "ColumnType":
         """
         spec, e.g. "VARCHAR(100) ASCII" is also available in
         exa.meta.input_columns[0].sql_type
         """
         sql_type = SqlType.from_string(spec)
-        column_class = cls.get_class(sql_type.name)
-        return column_class.from_sql(column_name, sql_type)
+        subclass = cls.get_class(sql_type.name)
+        return subclass.from_sql(sql_type)
+
+
+def make_column(name: str, type: ColumnType) -> "Column":
+    """
+    Instanciate a Column with name specified as a simple str, rather
+    than a ColumnName object.
+    """
+    from exasol.analytics.schema.column import Column
+
+    return Column(ColumnName(name), type)
 
 
 @dataclass(frozen=True, repr=True, eq=True)
-class BooleanColumn(Column):
+class BooleanColumn(ColumnType):
```

Comment:
if the super type is ColumType, I am not sure if BooleanColumn is an appropriate name , some for the other ones